browser/ui/webui/side_panel/customize_chrome/customize_toolbar/brave_action.h

```diff
@@ -69,14 +70,20 @@ inline constexpr BraveAction kShowVPNAction = {
     .icon = kLeoProductVpnIcon};
 #endif  // BUILDFLAG(ENABLE_BRAVE_VPN)
 
-inline constexpr auto kBraveActions = std::to_array<BraveAction>({
-    kShowSidePanelAction,
-    kShowWalletAction,
-    kShowAIChatAction,
+inline constexpr auto kBraveActions =
```

Comment:
Shouldn't we delete the `.id` field from `BraveAction`, if we are now indexing the actions with the id in this map?

Reply:
Oh, I see... let me think.

Reply:
Well, let's keep the field then, but then can change the map to this:

```cxx
    base::MakeFixedFlatMap<side_panel::customize_chrome::mojom::ActionId,
                           const BraveAction*>({
        {kShowSidePanelAction.id,
         &kShowSidePanelAction},
        {kShowWalletAction.id,
         &kShowWalletAction},
        {kShowAIChatAction.id,
         &kShowAIChatAction},
#if BUILDFLAG(ENABLE_BRAVE_VPN)
        {kShowVPNAction.id,
         &kShowVPNAction},
#endif  // BUILDFLAG(ENABLE_BRAVE_VPN)
    });
```